integration/config/configtx.yaml

```diff
@@ -23,11 +23,20 @@ Organizations:
         # Name can include alphanumeric characters as well as dots and dashes.
         Name: SampleOrg
 
+        # SkipAsForeign can be set to true for org definitions which are to be
+        # inherited from the orderer system channel during channel creation.  This
+        # is especially useful when an admin of a single org without access to the
+        # MSP directories of the other orgs wishes to create a channel.  Note
+        # this property must always be set to false for orgs included in block
+        # creation.
+        SkipAsForeign: false
+
         # ID is the key by which this org's MSP definition will be referenced.
         # ID can include alphanumeric characters as well as dots and dashes.
         ID: SampleOrg
 
         # MSPDir is the filesystem path which contains the MSP configuration.
+        #MSPDir: organizations/peerOrganizations/sampleorg.example.com/msp
```

Comment:
I've mentioned that in an earlier review but i guess it got shadowed by me adding the review and you pushing an overlapping update: this comment seems a superfluous difference to the sampleconfig version. Let's keep the files such that differences to the templates only contain essential differences, i.e., if i do a `diff -ur ../../hyperledger/fabric/sampleconfig/ integration/config/ ` i see only differences which matter (with the exception of this and below comment on orderer.example.com i think this is actually true now)